extensions/hibernate-envers/runtime/src/main/java/io/quarkus/hibernate/envers/HibernateEnversRecorder.java

```diff
@@ -32,43 +32,43 @@ private HibernateEnversIntegrationStaticInitListener(HibernateEnversBuildTimeCon
         @Override
         public void contributeBootProperties(BiConsumer<String, Object> propertyCollector) {
             var puConfig = buildTimeConfig.getAllPersistenceUnitConfigsAsMap().get(puName);
-            if (puConfig != null && puConfig.active.isPresent() && !puConfig.active.get()) {
+            if (puConfig == null || (puConfig.active.isPresent() && !puConfig.active.get())) {
                 propertyCollector.accept(EnversService.INTEGRATION_ENABLED, "false");
                 // Do not process other properties: Hibernate Envers is inactive anyway.
                 return;
             }
```

Comment:
We generally take the approach that extensions are activated automatically by default, as soon as they're in the classpath. That is, `active` is intended to default to `true`. So... why this change in behavior here?

If you just wanted to avoid a `NullPointerException`, I think this would be more correct:

```suggestion
            if (puConfig == null) {
                // Leave Envers unconfigured, but still activate it.
                return;
            }
            if (puConfig.active.isPresent() && !puConfig.active.get()) {
                propertyCollector.accept(EnversService.INTEGRATION_ENABLED, "false");
                // Do not process other properties: Hibernate Envers is inactive anyway.
                return;
            }
```